crates/shared/src/bad_token/token_owner_finder/blockscout.rs

```diff
@@ -22,6 +22,7 @@ impl BlockscoutTokenOwnerFinder {
             5 => "https://eth-goerli.blockscout.com/api",
             100 => "https://blockscout.com/xdai/mainnet/api",
             11155111 => "https://eth-sepolia.blockscout.com/api",
+            // TODO: should arb1 be here too?
```

Comment:
`https://arbitrum.blockscout.com/api` should be probably used for the arb1 since a tokenOwnerFinder is configured: https://github.com/cowprotocol/infrastructure/pull/1535/files#diff-9056c74c0da2da5bf9f902937af7d93fc926f33b2142e30b154dbbea16a4e825R91

Reply:
Thank you!

https://github.com/cowprotocol/services/pull/2757/commits/f753b0eb48bc885ab75f2065bb92ff90f5d9b4f2

Reply:
Nice, let's wait for Martin's approval before merging it.

Reply:
Ops, sorry. I turned on the auto-merge and your approval was enough for it to trigger 😬 

@MartinquaXD let me know whether you want me to revert or do any changes

Reply:
No problem. Thanks for the PR. 🙌